src/components/ec-file-upload/ec-file-upload.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-file-upload` : 'ec-file-upload'">
+    <label
+      v-if="label || note"
+      class="ec-file-upload__label"
+    >
+      <span
+        v-if="label"
+        data-test="ec-file-upload__label-text"
+        class="ec-file-upload__label-text"
+      >{{ label }}</span>
+
+      <span
+        v-if="note"
+        data-test="ec-file-upload__label-note"
+        class="ec-file-upload__label-note"
+      >{{ note }}</span>
+    </label>
+
+    <ec-file-dropzone
+      data-test="ec-file-upload__dropzone"
+      :is-disabled="isDisabled"
+      @change="onChange"
+    >
+      <slot
+        name="title"
+        slot="title"
+      />
+
+      <slot
+        name="subtitle"
+        slot="subtitle"
+      />
+    </ec-file-dropzone>
+
+    <div
+      data-test="ec-file-upload__help-text"
+      class="ec-file-upload__help-text"
+      :class="{ 'ec-file-upload__help-text--is-disabled': isDisabled }"
```

Comment:
We have a misalignment here. The class used is `ec-file-upload__help-text--is-disabled`, however, within the `<style>` block the defined is `&--disabled` (without the `is-` prefix). I actually think you can remove both pieces of code: this line, and the `&--disabled` style block, since it is not being applied right now and it looks good. In fact, if you apply the defined style the text background is displayed in grey.